streamx-console/streamx-console-service/src/main/java/com/streamxhub/streamx/console/core/controller/ProjectController.java

```diff
@@ -54,6 +54,18 @@ public RestResponse create(Project project) {
         return projectService.create(project);
     }
 
+    @PostMapping("update")
+    @RequiresPermissions("project:update")
+    public RestResponse update(Project project) {
+        boolean update = projectService.update(project);
+        return RestResponse.create().data(update);
+    }
+
+    @PostMapping("get")
+    public RestResponse get(Long id) {
+        return RestResponse.create().data(projectService.getProject(id));
```

Comment:
projectService.getById(id)


there are already methods to query objects according to the primary key. There is no need to rewrite another method